pkgs/development/python-modules/flask-paranoid/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, flask
+, pytest
+, pytest-runner
```

```suggestion
, flask
, pytestCheckHook
```
Please remove unused inputs.